apps/modernization-ui/src/generated/schema.graphqls

```diff
@@ -220,6 +223,7 @@ enum InvestigationEventDateType {
 }
 
 enum InvestigationStatus {
+    UNASSIGNED
```

Comment:
I don't believe `UNASSIGNED` applies to Investigation status. I think they can only be `OPEN` or `CLOSED`